.travis.yml

```diff
@@ -76,7 +76,7 @@ before_install:
   - if [ -x .travis/before_install_${TARGET}.sh ]; then .travis/before_install_${TARGET}.sh; fi;
 
 install:
-  - if [ -x .travis/install_${TARGET}.sh ]; then .travis/install_${TARGET}.sh; fi;
+  - if [ -x .travis/install_${TARGET}.sh ]; then travis_wait 40 .travis/install_${TARGET}.sh; fi;
```

Comment:
:thinking: ok so now Composer is longer?

Reply:
Yes, it's the same on local machine